app/views/ReportView/components/PathwayAnalysis/index.tsx

```diff
@@ -49,7 +49,7 @@ const PathwayAnalysis = ({
             `/reports/${report.ident}/image/retrieve/pathwayAnalysis.legend`,
             {},
           ).request();
-          setLegend(legendResp['pathwayAnalysis.legend']);
+          setLegend(legendResp[0]);
```

Comment:
is there a record I can see this in action?

Reply:
Any report on the page `/pathway-analysis` should show this

Reply:
looks to be an array return ... since there's no custom legends and this returns the expected type it's fine for now